libr/core/anal.c

```diff
@@ -239,6 +239,16 @@ R_API void r_core_anal_autoname_all_fcns(RCore *core) {
 		}
 	}
 }
+
+static bool blacklisted_word(char* name) {
+	const char * list[] = {"__stack_chk_guard", "__stderrp", "__stdinp", "__stdoutp", "_DefaultRuneLocale"};
```

Comment:
missing space after { and before }. putting this in 3 lines will be more readable and honor the 78 column rule:
```
const char * list[] = {
      "__stack_chk_guard", "__stderrp", "__stdinp", "__stdoutp", "_DefaultRuneLocale"
};

```